README.md

```diff
@@ -428,6 +428,21 @@ Finally, restart Postfix
 sudo systemctl restart postfix
 ```
 
+#### [Optional] Enable TLS on Postfix submission
+
+In case your Postfix server is on another server, it's recommended to enable TLS on Postfix submission to 
+secure the connection between SimpleLogin email handler and Postfix.
+
+This can be enabled by adding those lines at the end of `/etc/postfix/master.cf`
+
+```
+submission inet n       -       y       -       -       smtpd
```

Comment:
According to this [article](https://www.faqforge.com/linux/how-to-enable-port-587-submission-in-postfix/) you only need to uncomment this line or add it if it doesn't exist. Not sure what the best practice would be for the lines below.

Reply:
Some Postfix default config file uses a slightly different version of these lines. So I think it's less error-prone to add new lines instead of uncommenting existing lines.